instrumentation/net/http/httptrace/otelhttptrace/clienttrace.go

```diff
@@ -151,7 +163,8 @@ func NewClientTrace(ctx context.Context, opts ...ClientTraceOption) *httptrace.C
 		opt.apply(ct)
 	}
 
-	ct.tr = otel.GetTracerProvider().Tracer(
+	ct.tracerProvider = otel.GetTracerProvider()
```

Comment:
```suggestion
```

This should be set in the initialization of `ct` above, before the options are applied.  The goal is to make `otel.GetTracerProvider()` the default that will be used in the absence of any option that changes it, but to allow it to be overridden by an option.

Reply:
Ok, will make the changes to that. Should `ct.tr` also be set in the initialization of `ct` then? Or should it just be left where it originally was?

Reply:
`ct.tr` doesn't need to be initialized, it will always be set here to a new `Tracer` created from `ct.tracerProvider` and there are no options for setting it explicitly.

Reply:
Great, thanks!